sphinx_needs/directives/needimport.py

```diff
@@ -146,6 +143,13 @@ def run(self) -> Sequence[nodes.Node]:
 
         needs_config = NeedsSphinxConfig(self.config)
         data = needs_import_list["versions"][version]
+
+        if ids := self.options.get("ids"):
+            id_list = [i.strip() for i in ids.split(",") if i.strip()]
```

Comment:
Can IDs contain a dynamic function, so you would have to use the 'split with dyn' function?

Reply:
no definitely not, no dynamic functions here; this is intended for fast filtering